src/test/csi-qingcloud/templates/vpa.yaml

```diff
@@ -0,0 +1,58 @@
+
+{{ if .Values.enableVPA }}
+---
+apiVersion: "autoscaling.k8s.io/v1"
+kind: VerticalPodAutoscaler
+metadata:
+  name: csi-vpa
```

Comment:
`csi-qingcloud-vpa` may be more clear.

Reply:
How about rename it to `csi-qingcloud-controller-vpa` and add `csi-qingcloud-node-vpa` ?

Reply:
csi-qingcloud-controller-vpa is good